### PR TITLE
dd-2286 - fix conversion issues in the Updating guide

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR fixes the formatting issues in the Updating guide.

### CODE REVIEW

File: downstream/modules/platform/con-update-aap-planning.adoc [hunk 1]
```diff
@@ -0,0 +1,47 @@
+[id="con-update-aap-planning"]
+
+== Update planning
+
+Before you begin the update process, review the following considerations to plan and prepare your {PlatformNameShort} deployment:
+
+* Even if you have a valid license from a previous version, you must provide your credentials or a subscription manifest upon upgrading to the latest version of {PlatformNameShort}. For more information, see link:docs.redhat.com/en/documentation/{PlatformName}/2.5/html/access_management_and_authentication/assembly-gateway-licensing#proc-attaching-subscriptions[Attaching your {PlatformName} subscription].
```
Comment (hherbly) [inline downstream/modules/platform/con-update-aap-planning.adoc:7]: Should we use a link attribute here? I think it'd be: 
link:{URLCentralAuth}/assembly-gateway-licensing#proc-attaching-subscriptions[Attaching your {PlatformName} subscription]

File: downstream/modules/platform/con-update-aap-planning.adoc [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+* Clustered upgrades require special attention to instance and instance groups before upgrading. Ensure you capture your inventory or instance group details before upgrading. For more information, see link:docs.redhat.com/en/documentation/{PlatformName}/2.5/html-single/configuring_automation_execution/index#controller-clustering[Clustering].
```
Comment (hherbly) [inline downstream/modules/platform/con-update-aap-planning.adoc:9]: This link could be: 
link:{URLControllerAdminGuide}/index#controller-clustering[Clustering]

File: downstream/modules/platform/con-update-aap-planning.adoc [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+* If you are currently running {EDAcontroller}, it is recommended that you disable all rulebook activations before upgrading to ensure that only new activations run after the upgrade process has completed. This prevents possibilities of orphaned containers running activations from the previous version. For more information, see link:docs.redhat.com/en/documentation/{PlatformName}/2.5/html-single/using_automation_decisions/index#eda-enable-rulebook-activations[Enabling and disabling rulebook activations].
```
Comment (hherbly) [inline downstream/modules/platform/con-update-aap-planning.adoc:11]: suggested update to link:
link:{URLEDAUserGuide}/eda-rulebook-activations#eda-enable-rulebook-activations[Enabling and disabling rulebook activations]

File: downstream/modules/platform/con-update-aap-planning.adoc [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+== Backing up RPM-based Ansible Automation Platform
+
+Back up an existing {PlatformNameShort} instance by running the `setup.sh` script with the `backup_dir` flag, which saves the content and configuration of your current environment:
+
+. Go to your Ansible Automation Platform installation directory.
```
Comment (hherbly) [inline downstream/modules/platform/con-update-aap-planning.adoc:17]: Use the attribute here: {PlatformNameShort}

File: downstream/modules/platform/con-update-aap-planning.adoc [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+[literal, options="nowrap" subs="+attributes"]
+----
+$ ./setup.sh -e ‘backup_dir=/ansible/mybackup’ -e ‘use_compression=True’ @credentials.yml -b
++
+* `backup_dir` specifies a directory to save your backup to.
++
+* `@credentials.yml` passes the password variables and their values that are encrypted by `ansible-vault`.
+
+With a successful backup, a backup file is created at `/ansible/mybackup/<platform_installation_directory_name>.tar.gz`.
+
```
Comment (hherbly) [inline downstream/modules/platform/con-update-aap-planning.adoc:30]: Looks like the end of the code block should be right here, in line 30. As it is, it encompasses the rest of the module.

File: downstream/modules/platform/con-update-aap-planning.adoc [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+Back up an existing {PlatformNameShort} instance by running the `setup.sh` script with the `backup_dir` flag, which saves the content and configuration of your current environment:
+
+. Go to your Ansible Automation Platform installation directory.
+
+. Run the setup.sh script following the example below:
++
+[literal, options="nowrap" subs="+attributes"]
+----
+$ ./setup.sh -e ‘backup_dir=/ansible/mybackup’ -e ‘use_compression=True’ @credentials.yml -b
++
```
Comment (hherbly) [inline downstream/modules/platform/con-update-aap-planning.adoc:24]: Looks like the end of the code block should be right here, in line 24. As it is, it encompasses the rest of the module.

File: downstream/modules/platform/proc-backup-aap-rpm.adoc [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
++
+* `backup_dir` specifies a directory to save your backup to.
++
+* `@credentials.yml` passes the password variables and their values that are encrypted by `ansible-vault`.
+
+With a successful backup, a backup file is created at `/ansible/mybackup/<platform_installation_directory_name>.tar.gz`.
+
+*Additional resources*
+
+* For more information about backup and restore, see link:{URLControllerAdminGuide}/controller-backup-and-restore[Backup and restore] in _{TitleControllerAdminGuide}_.
```
Comment (hherbly) [inline downstream/modules/platform/proc-backup-aap-rpm.adoc:23]: backing up and restoring?

File: downstream/modules/platform/proc-inventory-file-setup-rpm.adoc [hunk 8]
```diff
@@ -0,0 +1,28 @@
+[id="inventory-file-setup-rpm"]
+
+= Setting up the inventory file
+
+Before upgrading your {PlatformName} installation, edit the `inventory file so that it matches your desired configuration. You can keep the same parameters from your existing Ansible Automation Platform deployment or you can modify the parameters to match any changes to your environment.
```
Comment (hherbly) [inline downstream/modules/platform/proc-inventory-file-setup-rpm.adoc:5]: missed attribute in the second sentence here - {PlatformNameShort}

File: downstream/modules/platform/proc-inventory-file-setup-rpm.adoc [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
++
+----
+$ cd ansible-automation-platform-setup-<version_number>
+----
++
+. Update the `inventory` file to match your desired configuration. You can use the same inventory file from an existing {PlatformNameShort} installation if there are no changes to the environment.
+
+[NOTE]
+====
+Provide a reachable IP address or fully qualified domain name (FQDN) for all hosts to ensure that users can synchronize and install content from Ansible automation hub from a different node. Do not use localhost. If localhost is used, the upgrade will be stopped as part of preflight checks.
```
Comment (hherbly) [inline downstream/modules/platform/proc-inventory-file-setup-rpm.adoc:27]: Missed attribute for automation hub. This one should do it {HubNameMain}

File: downstream/modules/platform/proc-update-aap-container.adoc [hunk 10]
```diff
@@ -0,0 +1,51 @@
+[id="proc-update-aap-container"]
+
+= Updating container-based {PlatformNameShort}
+
+Perform a patch update for a container-based installation of Ansible Automation Platform.
```
Comment (hherbly) [inline downstream/modules/platform/proc-update-aap-container.adoc:5]: missed attribute: {PlatformNameShort}

File: downstream/modules/platform/proc-update-aap-container.adoc [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+
+.Prerequisites
+
+* Reviewed the release notes for the associated patch release.
+
+* Created a backup of your {PlatformNameShort} deployment. For more information, see xref:proc-backup-aap-container[Backing up controller-based {PlatformNameShort}].
+
+.Procedure
+
+. Download the latest version of the containerized installer from the link:https://access.redhat.com/downloads/content/480/ver=2.5/rhel---9/2.5/x86_64/product-software[{PlatformNameShort} download] page.
```
Comment (hherbly) [inline downstream/modules/platform/proc-update-aap-container.adoc:15]: There's an attribute for the download page. It's {PlatformDownloadUrl} so I think you'd format it this way:
link:{PlatformDownloadUrl}[{PlatformNameShort} download]

File: downstream/modules/platform/proc-update-aap-container.adoc [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
++
+. Edit the `inventory` file so that it matches your desired configuration. You can keep the same parameters from your existing {PlatformNameShort} deployment or you can modify the parameters to match any changes to your environment.
+
+. Run the `install` command:
++
+----
+$ ansible-playbook -i inventory ansible.containerized_installer.install
+----
++
+* If your privilege escalation requires a password to be entered, append -K to the command. You will then be prompted for the BECOME password.
```
Comment (hherbly) [inline downstream/modules/platform/proc-update-aap-container.adoc:47]: Looks like there's a repeated bullet point here. I'd remove the first one.

File: downstream/modules/platform/proc-update-aap-container.adoc [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+* If your privilege escalation requires a password to be entered, append `-K` to the command. You will then be prompted for the `BECOME password.
```
Comment (hherbly) [inline downstream/modules/platform/proc-update-aap-container.adoc:48]: Missed quotation mark after BECOME

File: downstream/modules/platform/proc-update-aap-rpm.adoc [hunk 14]
```diff
@@ -0,0 +1,131 @@
+[id="proc-update-aap-rpm"]
+
+== Update planning
+
+Before you begin the update process, review the following considerations to plan and prepare your {PlatformNameShort} deployment:
+
+* Even if you have a valid license from a previous version, you must provide your credentials or a subscription manifest upon upgrading to the latest version of {PlatformNameShort}. For more information, see link:docs.redhat.com/en/documentation/{PlatformName}/2.5/html/access_management_and_authentication/assembly-gateway-licensing#proc-attaching-subscriptions[Attaching your {PlatformName} subscription].
```
Comment (hherbly) [inline downstream/modules/platform/proc-update-aap-rpm.adoc:7]: alternately formatted link: 
link:{URLCentralAuth}/assembly-gateway-licensing#proc-attaching-subscriptions[Attaching your {PlatformName} subscription]

File: downstream/modules/platform/proc-update-aap-rpm.adoc [hunk 15]
```diff
@@ -0,0 +1,131 @@
...
+
+* Clustered upgrades require special attention to instance and instance groups before upgrading. Ensure you capture your inventory or instance group details before upgrading. For more information, see link:docs.redhat.com/en/documentation/{PlatformName}/2.5/html-single/configuring_automation_execution/index#controller-clustering[Clustering].
```
Comment (hherbly) [inline downstream/modules/platform/proc-update-aap-rpm.adoc:9]: link:{URLControllerAdminGuide}/controller-clustering[Clustering]

File: downstream/modules/platform/proc-update-aap-rpm.adoc [hunk 16]
```diff
@@ -0,0 +1,131 @@
...
+
+* If you are currently running {EDAcontroller}, it is recommended that you disable all rulebook activations before upgrading to ensure that only new activations run after the upgrade process has completed. This prevents possibilities of orphaned containers running activations from the previous version. For more information, see link:docs.redhat.com/en/documentation/{PlatformName}/2.5/html-single/using_automation_decisions/index#eda-enable-rulebook-activations[Enabling and disabling rulebook activations].
```
Comment (hherbly) [inline downstream/modules/platform/proc-update-aap-rpm.adoc:11]: link:{URLEDAUserGuide}/eda-rulebook-activations#eda-enable-rulebook-activations[Enabling and disabling rulebook activations]

File: downstream/modules/platform/proc-update-aap-rpm.adoc [hunk 17]
```diff
@@ -0,0 +1,131 @@
...
+
+A valid Red Hat customer account is required to access {PlatformName} installer downloads on the Red Hat Customer Portal.
+
+== Installing with internet access
+
+*Tarball install*
+
+.Procedure
+
+. Navigate to the link:access.redhat.com/downloads/content/480/ver=2.5/rhel---9/2.5/x86_64/product-software[{PlatformName} download download] page.
```
Comment (hherbly) [inline downstream/modules/platform/proc-update-aap-rpm.adoc:26]: Two "download"s in the link title here. 
You could also use:
link:{PlatformDownloadUrl}[{PlatformNameShort} download]

File: downstream/modules/platform/con-update-aap-planning.adoc (hunk 1 above)
Comment (dcdacosta) [reply downstream/modules/platform/con-update-aap-planning.adoc]: File no longer used. It's been removed.

File: downstream/modules/platform/con-update-aap-planning.adoc (hunk 2 above)
Comment (dcdacosta) [reply downstream/modules/platform/con-update-aap-planning.adoc]: File no longer used. It has been removed.

File: downstream/modules/platform/con-update-aap-planning.adoc (hunk 3 above)
Comment (dcdacosta) [reply downstream/modules/platform/con-update-aap-planning.adoc]: File no longer used. It has been removed.

File: downstream/modules/platform/con-update-aap-planning.adoc (hunk 4 above)
Comment (dcdacosta) [reply downstream/modules/platform/con-update-aap-planning.adoc]: File no longer used. It has been removed.

File: downstream/modules/platform/con-update-aap-planning.adoc (hunk 6 above)
Comment (dcdacosta) [reply downstream/modules/platform/con-update-aap-planning.adoc]: File no longer used. It has been removed.

File: downstream/modules/platform/proc-backup-aap-rpm.adoc (hunk 7 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-backup-aap-rpm.adoc]: Fixed.

File: downstream/modules/platform/proc-inventory-file-setup-rpm.adoc (hunk 8 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-inventory-file-setup-rpm.adoc]: Fixed.

File: downstream/modules/platform/proc-inventory-file-setup-rpm.adoc (hunk 9 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-inventory-file-setup-rpm.adoc]: Fixed.

File: downstream/modules/platform/proc-update-aap-container.adoc (hunk 10 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-update-aap-container.adoc]: Fixed.

File: downstream/modules/platform/proc-update-aap-container.adoc (hunk 11 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-update-aap-container.adoc]: Fixed.

File: downstream/modules/platform/proc-update-aap-container.adoc (hunk 12 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-update-aap-container.adoc]: Fixed.

File: downstream/modules/platform/proc-update-aap-container.adoc (hunk 13 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-update-aap-container.adoc]: Fixed.

File: downstream/modules/platform/proc-update-aap-rpm.adoc (hunk 14 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-update-aap-rpm.adoc]: File no longer used. It has been removed.

File: downstream/modules/platform/proc-update-aap-rpm.adoc (hunk 15 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-update-aap-rpm.adoc]: File no longer used. It has been removed.

File: downstream/modules/platform/proc-update-aap-rpm.adoc (hunk 16 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-update-aap-rpm.adoc]: File no longer used. It has been removed.

File: downstream/modules/platform/proc-update-aap-rpm.adoc (hunk 17 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-update-aap-rpm.adoc]: File no longer used. It has been removed.

File: downstream/modules/platform/proc-update-aap-container.adoc [hunk 18]
```diff
@@ -0,0 +1,50 @@
...
+
+= Updating container-based {PlatformNameShort}
+
+Perform a patch update for a container-based installation of {PlatformNameShort}.
+
+.Prerequisites
+
+* Reviewed the release notes for the associated patch release.
+
+* Created a backup of your {PlatformNameShort} deployment. For more information, see xref:proc-backup-aap-container[Backing up controller-based {PlatformNameShort}].
```
Comment (hherbly) [inline downstream/modules/platform/proc-update-aap-container.adoc:11]: I'd add "You have:" right below the heading to make the bullet points read like complete sentences

Comment (dcdacosta) [reply downstream/modules/platform/proc-update-aap-container.adoc:13]: Done.

**Review by hherbly — APPROVED** [review]
LGTM!

**Pull request merged**